packages/style-engine/class-wp-style-engine-css-declarations.php

```diff
@@ -123,10 +120,7 @@ public function get_declarations() {
 	 * @return string The filtered declaration as a single string.
 	 */
 	protected static function filter_declaration( $property, $value, $spacer = '' ) {
-		if ( isset( $property ) && isset( $value ) ) {
-			return safecss_filter_attr( "{$property}:{$spacer}{$value}" );
-		}
-		return '';
+		return safecss_filter_attr( "{$property}:{$spacer}{$value}" );
```

Comment:
`$property` and `$value` will always be set, they are mandatory params of the method.

Reply:
Are there checks somewhere else?

Reply:
Yes! We check the property and value when adding them to the object, so we already know they're there 😉